galaxykit/registries.py

```diff
@@ -8,7 +9,10 @@ def get_registry_pk(client, name):
     user_url = f"_ui/v1/execution-environments/registries/?name={name}"
     resp = client.get(user_url)
     if resp["data"]:
-        return resp["data"][0]["id"]
+        if parse_version(client.server_version) > parse_version("4.6.2"):
```

Comment:
Should this be `... > parse_version("4.7.0dev")` so it doesn't do the wrong thing when we release 4.6.3?

Reply:
ah good catch, I fixed that elsewhere but missed this spot!